src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,65 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesNumber = Math.ceil(total / perPage);
+  const paginated = getNumbers(1, pagesNumber);
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 && ('disabled')}`}>
```

Comment:
pls make use of `classNames` library here
Avoid using boolean values in template literals. It's better to use conditional rendering.
```suggestion
      <li className=classNames('page-item', {
          disabled: currentPage === 1,
        }>
```